### PR TITLE
add codespell workflow (#1)

### PR DESCRIPTION
**Pull request by dwerder**
* add codespell workflow
* add chart-testing action
* add documentation and PR template

Signed-off-by: Daniel Werdermann <daniel.werdermann@gmail.com>

### CODE REVIEW

File: .github/workflows/lint-test.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
+        if: steps.list-changed.outputs.changed == 'true'
+
+      - name: Run chart-testing (install)
+        run: ct install --config ct.yaml
```
Comment (haarchri) [inline .github/workflows/lint-test.yaml:42]: please add newline

File: ct.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+target-branch: main
+chart-dirs:
+  - charts
```
Comment (haarchri) [inline ct.yaml:5]: please add newline

**Review by pierluigilenoci — APPROVED** [review]

**Review by haarchri — CHANGES_REQUESTED** [review]
LGTM - please check the newlines

File: .github/workflows/lint-test.yaml (hunk 1 above)
Comment (dwerder) [reply .github/workflows/lint-test.yaml:42]: done

**Comment by haarchri** [comment]
thanks LGTM

**Pull request merged**